ts/parts-more/BubbleSeries.ts

```diff
@@ -41,6 +41,7 @@ declare global {
             minPxSize?: BubbleSeries['minPxSize'];
             specialGroup?: BubbleSeries['specialGroup'];
             zData?: BubbleSeries['zData'];
+            yData?: BubbleSeries['yData'];
```

Comment:
You should defined yData as a permanent property in the BubbleSeries class (like zData), otherwise you are here actually referencing to the base Series class, because there is no further clarification in BubbleSeries.

Reply:
Looks good!